test/options.test.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 const t = require('tap')
-require('./helper').payloadMethod('options', t)
+// require('./helper').payloadMethod('options', t)
```

Comment:
We are almost there!!
Why this is commented?

Reply:
Oioioioioi, sorry and thanks for reviewing!